apps/gallery/index.html

```diff
@@ -105,7 +105,7 @@ <h1 id="pick-header-title" data-l10n-id="pickoneimage2"></h1>
     </section>
 
     <section role="region" id="fullscreen-view" class="skin-dark">
-      <gaia-header id="fullscreen-toolbar-header" action="back" class="large-only">
+      <gaia-header id="fullscreen-toolbar-header" action="back" class="large-only" no-font-fit>
```

Comment:
This one has an action attribute, so why doesn't it have title-start="50" like the other headers with action buttons?
The "large-only" class means that this header is only for the no-longer-supported tablet form factor, so we don't really have any way of testing the change here.

Given that this title bar will be hidden, will it still run the font-fit logic on startup? That is, do we need the no-font-fit logic here, or can we leave this change out?

I'd prefer to leave it out (since it is untestable and unsupported) if possible.